engine/access/rpc/backend/backend_transactions.go

```diff
@@ -246,14 +269,98 @@ func (b *backendTransactions) GetTransactionResult(
 	}
 
 	return &access.TransactionResult{
-		Status:       txStatus,
-		StatusCode:   uint(statusCode),
-		Events:       events,
-		ErrorMessage: txError,
-		BlockID:      blockID,
+		Status:        txStatus,
+		StatusCode:    uint(statusCode),
+		Events:        events,
+		ErrorMessage:  txError,
+		BlockID:       blockID,
+		TransactionID: txID,
```

Comment:
should we include the `CollectionID` here too for consistency?

Reply:
CollectionID is ambiguous for this method